.mvn/maven.config

```
@@ -1,4 +1,4 @@
--Drevision=1.11.0-SNAPSHOT
+-Drevision=1.11.0
```

```suggestion
-Drevision=1.12.0-SNAPSHOT
```

as long as this merges into `main`